src/Machine.js

```diff
@@ -151,6 +151,15 @@ class Machine {
     this._globalContext = {}; // global context
   }
 
+  /**
+   * Format path
+   * @private
+   */
+  _formatPath(path) {
+    let _path = path.replace(/\\/g, '/');
+    return _path ? _path + '/' : '';	  
```

Comment:
Please use system API to deal with file paths https://www.w3schools.com/nodejs/met_path_join.asp

Reply:
path.join and path.normalize is used.